components/brave_shields/core/test/brave_shields_utils_unittest.cc

```diff
@@ -769,6 +770,58 @@ TEST_F(BraveShieldsUtilTest, SetFingerprintingControlType_ForOrigin) {
   EXPECT_EQ(ControlType::DEFAULT, type);
 }
 
+/* REDUCE LANGUAGE */
+TEST_F(BraveShieldsUtilTest, ReduceLanguageEnabledForProfileFeatureDisabled) {
+  base::test::ScopedFeatureList scoped_feature_list;
+  scoped_feature_list.InitAndDisableFeature(
+      brave_shields::features::kBraveReduceLanguage);
+  EXPECT_FALSE(
+      brave_shields::IsReduceLanguageEnabledForProfile(profile()->GetPrefs()));
+}
+
+TEST_F(BraveShieldsUtilTest, IsReduceLanguageEnabledForProfile) {
+  base::test::ScopedFeatureList scoped_feature_list;
+  scoped_feature_list.InitAndEnableFeature(
+      brave_shields::features::kBraveReduceLanguage);
+
+  EXPECT_TRUE(
+      brave_shields::IsReduceLanguageEnabledForProfile(profile()->GetPrefs()));
+  profile()->GetPrefs()->SetBoolean(
+      brave_shields::prefs::kReduceLanguageEnabled, false);
+  EXPECT_FALSE(
+      brave_shields::IsReduceLanguageEnabledForProfile(profile()->GetPrefs()));
+  profile()->GetPrefs()->SetBoolean(
+      brave_shields::prefs::kReduceLanguageEnabled, true);
+  EXPECT_TRUE(
+      brave_shields::IsReduceLanguageEnabledForProfile(profile()->GetPrefs()));
+}
+
+TEST_F(BraveShieldsUtilTest, SetManagedIsReduceLanguageEnabledForProfile) {
```

Comment:
removing this unit test as it checks general managed functionality for `brave_shields::prefs::kReduceLanguageEnabled` preference